### PR TITLE
Adding some info to read when websocket connections fails

### PR DESCRIPTION
**Pull request by hrigner**
Will fix the failing tests when I have some positive feedback about this.. :)

### CODE REVIEW

File: src/components/splash.jsx [hunk 1]
```diff
@@ -45,10 +47,53 @@ export default function Splash({
...
+        For connecting to a Qlik Sense Enterprise on Windows using default virtual proxy, the websocket URL will
+        be
+        {' '}
+        <strong>wss://&lt;sense-host.com&gt;/app/&lt;app-GUID&gt;</strong>
+. If using a custom virtual proxy the
+        websocket URL will need to reflect this. Also note that for the Sense Proxy to allow sessions from catwalk,
+        catwalk needs to be whitelisted in Qlik Sense Enterprise.
+      </p>
+      <p>
+        Make sure that you are logged in to Qlik Sense in another browser tab/window. This is needed in order for catwalk
```
Comment (peol) [inline src/components/splash.jsx]: Maybe clarify this a bit? catwalk doesn't use the cookie, the browser will use the session cookie directly towards their deployment, nothing passes through the catwalk logic :)

File: src/components/splash.jsx [hunk 2]
```diff
@@ -45,10 +47,53 @@ export default function Splash({
...
+      <h3>Engine running in Docker container</h3>
+      <p>
+        For connecting to an Engine running in a Docker container with a mounted data volume containing the app and starting
+        engine with
+        {' '}
+        <strong>DocumentDirectory=&lt;data-folder&gt;</strong>
+, the websocket URL will
+        be
+        {' '}
+        <strong>ws://&lt;host&gt;:&lt;port&gt;/&lt;data-folder/&lt;app&gt;</strong>
```
[the remark this answers is not in the document]
Comment (hrigner) [reply src/components/splash.jsx]: Ah, I didn't know that. Interesting :)

File: src/components/splash.jsx (hunk 1 above)
Comment (hrigner) [reply src/components/splash.jsx]: yepp, true!

File: src/components/splash.jsx [hunk 3]
```diff
@@ -45,10 +47,53 @@ export default function Splash({
     </form>
   );
 
+  const learnMore = (
+    <Collapsible trigger="Learn more">
+      <h3>General</h3>
+      <p>Make sure that you have the right privileges to list apps or view the app.</p>
+      <h3>Engine running in Docker container</h3>
+      <p>
+        For connecting to an Engine running in a Docker container with a mounted data volume containing the app and starting
```
Comment (sublibra) [inline src/components/splash.jsx]: Long sentence. Consider adding a comma. E.g.  
For connecting to an Engine running in a docker container, with a mounted...

File: src/components/splash.jsx (hunk 2 above)
Comment (sublibra) [inline src/components/splash.jsx]: You are using `<app`> here and `<app-name>` further down.

File: src/components/splash.jsx [hunk 4]
```diff
@@ -56,38 +56,37 @@ export default function Splash({
...
       <h3>Qlik Sense Enterprise on Windows</h3>
       <p>
-        For connecting to Qlik Sense Enterprise on Windows using default virtual proxy, the websocket URL will
+        For connecting to Qlik Sense Enterprise on Windows, the websocket URL will
         be
         {' '}
-        <strong>wss://&lt;sense-host.com&gt;/app/&lt;app-GUID&gt;</strong>
-. If using a custom virtual proxy the
-        websocket URL will need to reflect this. Also note that for the Sense Proxy to allow sessions from catwalk,
+        <code>wss://&lt;sense-host.com&gt;/&lt;virtual-proxy-prefix&gt;/app/&lt;app-GUID&gt;</code>
```
Comment (axelssonHakan) [inline src/components/splash.jsx]: 👍

**Pull request merged**